src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,17 @@
+.roboto-regular {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.roboto-medium {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
```

Comment:
They are not mixins, just classes. And it doesn't make much sense as `Roboto` is a main font so you just set it to html or body tag and just change font weight when you need